### PR TITLE
Docs : missing part for start node and whitespace

### PR DESCRIPTION
**Pull request by spacegangster**
mostly whitespace

### CODE REVIEW

File: docs/src/docs/examples.clj [hunk 1]
```diff
@@ -12,11 +12,12 @@
 
 ;; tag::start-standalone-node[]
 (defn start-standalone-node ^crux.api.ICruxAPI [storage-dir]
-  (crux/start-node {:crux.node/topology '[crux.standalone/topology]
```
Comment (spacegangster) [inline docs/src/docs/examples.clj]: so the vector form doesn't conform the actual spec

File: docs/src/docs/examples.clj [hunk 2]
```diff
@@ -12,11 +12,12 @@
...
+  (crux/start-node {:crux.node/topology 'crux.standalone/topology
                     :crux.node/kv-store 'crux.kv.memdb/kv
                     :crux.kv/db-dir (str (io/file storage-dir "db"))
                     :crux.standalone/event-log-kv-store 'crux.kv.memdb/kv
                     :crux.standalone/event-log-dir (str (io/file storage-dir "event-log"))}))
+(def node (start-standalone-node "crux-store"))
```
Comment (spacegangster) [inline docs/src/docs/examples.clj]: so you can just copy and paste

**Comment by spacegangster** [comment]
another thought was, from a canary developer's perspective, would be cool to have a ready scenario for a canary deployment, so to prioritise rocks-db usage. I could _try_ to bring that into docs with a PR if you agree.

File: docs/src/docs/examples.clj (hunk 1 above)
Comment (danmason) [reply docs/src/docs/examples.clj]: This was bought in by something currently awaiting release - #589 (The docs currently point to master, so things like this pop up from time to time. We want to address that, see #610)

Comment (spacegangster) [reply docs/src/docs/examples.clj]: Thanks! Yeah, I forgot that docs could point to unreleased changes.
#610 would be awesome

File: docs/src/docs/examples.clj (hunk 2 above)
Comment (jarohen) [reply docs/src/docs/examples.clj]: I'd come down on the side of not including these top-level side-effects in our examples, particularly given this opens a closable resource - it's ok for REPL usage but outside of this we should be encouraging users to use `with-open`, or a component-like framework that ensures resources are closed properly.

(also, for internal dev experience, the tests evaluate this namespace to ensure the examples still work - opening a node here would likely affect being able to run the tests multiple times)

Comment (spacegangster) [reply docs/src/docs/examples.clj]: @jarohen good, then two options are left – discard the line or wrap into a `comment` form

Comment (spacegangster) [reply docs/src/docs/examples.clj]: @jarohen I've left a commented out suggestion, but feel free to drop it

**Comment by jarohen** [comment]
thanks @spacegangster :smile:

**Pull request merged**